apps/crn-server/src/handlers/user/sync-active-campaign-contact.ts

```diff
@@ -76,22 +86,54 @@ const getContactPayload = async (
       },
       {
         field: fieldIdByTitle['Working Group']!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
-        value: user.workingGroups[0]?.name || '',
+        value: user.workingGroups.map((item) => item.name).join(', '),
       },
       {
         field: fieldIdByTitle['Interest Group']!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
-        value: user.interestGroups[0]?.name || '',
+        value: user.interestGroups.map((item) => item.name).join(', '),
       },
       {
         field: fieldIdByTitle.LinkedIn!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
         value: user.social?.linkedIn || '',
       },
+      {
+        field: fieldIdByTitle.Network!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
+        value: '||ASAP CRN||',
+      },
     ],
   };
 };
 
+export const updateContactLists = async (contactId: string) => {
+  const listIdByName = await getListIdByName(
+    activeCampaignAccount,
+    activeCampaignToken,
+  );
+
+  const masterListId = listIdByName['Master List']!; // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
either make sure that these aren't represented as generic string indexes or do a check each time you use them.